core/templates/dev/head/pages/library/library.html

```diff
@@ -137,7 +138,8 @@ <h2 ng-class="{'active': activeGroupIndex === $index}" class="oppia-library-grou
                                               num-views="tile.num_views"
                                               thumbnail-icon-url="tile.thumbnail_icon_url"
                                               thumbnail-bg-color="tile.thumbnail_bg_color"
-                                              class="protractor-test-exp-summary-tile">
+                                              class="protractor-test-exp-summary-tile"
+                                              show-learner-dashboard-icons="userIsLoggedIn">
```

Comment:
This still shouldn't be passed through. The individual tile should check whether the user is logged in by using e.g. GLOBALS.userIsLoggedIn, and act accordingly.

Reply:
Okay, I'll just change this. Should I keep the attribute the name still as `show-learner-dashboard-icons`?

Reply:
I think you want the "-if-possible" part I mentioned, for the given reason. Similar comments apply to all the other places where you do this.

Reply:
Okay.. cool!

Reply:
I have fixed this. Feels good to see so much modularity! :D